src/test/java/software/amazon/encryption/s3/S3EncryptionClientTest.java

```diff
@@ -872,6 +845,62 @@ public void s3EncryptionClientMixedCredentials() {
         kmsClient.close();
     }
 
+    @Test
+    public void s3EncryptionClientMixedCredentialsInstructionFile() {
```

Comment:
I'm not sure what the scope of this test is; it looks like both a failure and success test case?
Could you clarify what exactly is being tested in comments here?

Reply:
sure, I'll break it out into two separate tests